src/transformers/modeling_longformer.py

```diff
@@ -12,10 +12,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""PyTorch Longformer model. """
+"""Tensorflow Longformer model. """
```

Comment:
Pretty sure this wants to stay as PyTorch
```suggestion
"""PyTorch Longformer model. """
```